.github/workflows/optimize-checkstyle-check.yml

```diff
@@ -0,0 +1,70 @@
+# TODO: Remove this pipeline once Optimize inherits from zeebe-parent,
```

Comment:
what do you think about having this as part of the acceptance criteria of https://github.com/camunda/camunda/issues/22271?

Reply:
that's a very good idea.
I'll add it. We have more than one TODO like this.
Thanks